js/apps/admin-ui/src/user/UsersSection.tsx

```diff
@@ -236,59 +306,425 @@ export default function UsersSection() {
   //should *only* list users when no user federation is configured
   const listUsers = !(userStorage.length > 0);
 
-  const toolbar = (
-    <>
+  const onSearch = () => {
+    if (searchValue !== "") {
+      setSearchValue(searchValue);
+    } else {
+      setSearchValue("");
+    }
+    refresh();
+  };
+
+  const handleKeyDown = (e: KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === "Enter") {
+      onSearch();
+      refresh();
+    }
+  };
+
+  const createQString = (filters: UserAttribute[]) => {
```

Comment:
Can we name this function fully as well?
```suggestion
  const createQueryString = (filters: UserAttribute[]) => {
```

Reply:
Fixed